flusher.go

```diff
@@ -317,6 +331,19 @@ func (s *Server) flushForward(wms []WorkerMetrics) {
 	jsonMetrics := make([]samplers.JSONMetric, 0, jmLength)
 	exportStart := time.Now()
 	for _, wm := range wms {
+		for _, count := range wm.globalCounters {
+			jm, err := count.Export(interval)
+			if err != nil {
+				log.WithFields(logrus.Fields{
+					logrus.ErrorKey: err,
+					"type":          "counter",
+					"name":          count.Name,
+				}).Error("Could not export metric")
+				continue
+			}
+			// do I need to set jm.Type here?
```

Comment:
extraneous comment?


pretty sure the answer is 'no', but the tests can be the judge of that.